Problems/10866/ans_10866_YH.py

```diff
@@ -0,0 +1,90 @@
+#https://www.acmicpc.net/problem/10866
+import sys
+input = sys.stdin.readline
+
+class Node:
+    def __init__(self,data):
+        self.data = data
+        self.prev = None
+        self.next = None
+
+class Deque:
+    def __init__(self):
+        self.size = 0
+        self.head = None
+        self.tail = None
+    def Push_front(self,data):
+        self.size +=1
+        NewNode = Node(data)
+        if self.head == None:
+            self.head = self.tail = NewNode
+            return 
+        # NewNode.prev = self.head
+        # NewNode.next = self.head.next
+        self.head.prev = NewNode
+        NewNode.next = self.head
+        self.head = NewNode
+    def Push_back(self,data):
+        self.size +=1
+        NewNode = Node(data)
+        if self.head == None:
+            self.head = self.tail = NewNode
+            return  
+        # NewNode.prev = self.tail.prev
+        # NewNode.next = self.tail
+        self.tail.next = NewNode
+        NewNode.prev = self.tail
+        self.tail = NewNode
+    def Pop_front(self):
+        if not self.head:
```

Comment:
```suggestion
        if self.head is not None:
```
을 표현하려고 했던 것 같은데
PEP8을 공부해보셔... 가독성이 너무 낮군